.github/workflows/test.yml

```diff
@@ -87,6 +91,10 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 1
+    - name: Install cosign binary
+      uses: sigstore/cosign-installer@v1.3.1
+      with:
+        cosign-release: 'v1.3.1'
```

Comment:
No longer needed AFAICS

Reply:
> Compilation failing for Windows
> 
> ```
> # github.com/containerd/nerdctl/cmd/nerdctl [github.com/containerd/nerdctl/cmd/nerdctl.test]
> cmd\nerdctl\pull_test.go:64:9: undefined: newTestRegistry
> cmd\nerdctl\pull_test.go:91:9: undefined: newTestRegistry
> FAIL	github.com/containerd/nerdctl/cmd/nerdctl [build failed]
> FAIL
> ```

Changed the filename, this one should fix it:

```
R  cmd/nerdctl/pull_test.go -> cmd/nerdctl/pull_linux_test.go
```